core-bundle/src/Resources/contao/controllers/BackendIndex.php

```diff
@@ -107,7 +108,7 @@ public function run()
 		$objTemplate->messages = Message::generate();
 		$objTemplate->base = Environment::get('base');
 		$objTemplate->language = $GLOBALS['TL_LANGUAGE'];
-		$objTemplate->languages = System::getLanguages(true); // backwards compatibility
+		$objTemplate->languages = System::getContainer()->get(Locales::class)->getDisplayNames(System::getContainer()->getParameter('contao.locales'), null, true); // backwards compatibility
```

Comment:
I think we can safely remove this completely, since the login screen does not have a language selection anymore and the variable has only been here for a few years for BC reasons.

Reply:
Unrelated to this PR.